File: Art Bahia Android/app/src/main/java/fja/edu/com/artbahia/Models/UsuarioDTO.java
package fja.edu.com.artbahia.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class UsuarioDTO {
    public int Id;
    public String Login;
    public String Senha;
    public String Nome;
    public boolean IsArtista;

    public UsuarioDTO(){

    }

    public UsuarioDTO(int id, String login, String senha, String nome,  boolean isArtista){
        this.Id = id;
        this.Login = login;
        this.Senha = senha;
        this.Nome = nome;
        this.IsArtista = isArtista;
    }
}

File: ArtBahiaAPI/DL - Data Layer/Models/UsuarioDTO.cs
using Dapper.Contrib.Extensions;

namespace ArtBahiaAPI.Models {
	[Table("Usuario")]
	public class UsuarioDTO {
		[Key]
		public int Id { get; set; }
		public string Login { get; set; }
		public string Senha { get; set; }
		public string Nome { get; set; }
		public bool IsArtista { get; set; }
	}
}
